web/src/cypress/integration/schedules.ts

```diff
@@ -191,7 +191,7 @@ function testSchedules(screen: ScreenFormat): void {
         'From 12:00 AM to 8:00 PM EDT',
       )
 
-      cy.get('[data-cy="shift-details"]').trigger('mouseover')
```

Comment:
We should keep `trigger('mouseover')` here, `focus()` won't trigger the tooltip to show up